File: client-java/controller-api/src/main/java/org/evomaster/client/java/controller/api/dto/StringSpecializationInfoDto.java
package org.evomaster.client.java.controller.api.dto;

public class StringSpecializationInfoDto {

    public String stringSpecialization;

    public String value;

    public String type;

    public StringSpecializationInfoDto(){}

    public StringSpecializationInfoDto(String stringSpecialization, String value, String type) {
        this.stringSpecialization = stringSpecialization;
        this.value = value;
        this.type = type;
    }
}

File: client-dotnet/src/EvoMaster.Controller.Api/dto/StringSpecializationInfoDto.cs
namespace EvoMaster.Controller.Api {
    public class StringSpecializationInfoDto {
        public StringSpecializationInfoDto() { }

        public StringSpecializationInfoDto(string stringSpecialization, string value, string type) {
            this.StringSpecialization = stringSpecialization;
            this.Value = value;
            this.Type = type;
        }

        public string StringSpecialization { get; set; }

        public string Value { get; set; }

        public string Type { get; set; }
    }
}